IP_POLICY_GUIDANCE.md

```diff
@@ -0,0 +1,94 @@
+# OpenJS Foundation IP Policy Guidance
+
+The purpose of this guide is to make it straightforward for projects
+to follow the [OpenJS Foundation IP Policy][IP Policy].
+
+Of course, the source of truth remains the [IP Policy][].
+
+If in doubt about anything related to IP after reading this document,
+first read the [IP Policy][], and then [ask for help](#getting-help).
+
+## Basic requirements
+
+There are four key requirements for all OpenJS Foundation projects:
+
+1. [Licensing](#1-licensing)
+2. [Copyright notices](#2-copyright-notices)
+3. Displaying the [standard website footer](#3-standard-website-footer) on the project's website
+4. [Adopting the DCO or a CLA](#4-adopting-the-dco-or-a-cla)
+
+### 1. Licensing
+
+Code may be licensed under:
+  * [Apache, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+  * [MIT](https://opensource.org/licenses/MIT)
+  * [2-Clause BSD](https://opensource.org/licenses/BSD-2-Clause)
+  * [3-Clause BSD](https://opensource.org/licenses/BSD-3-Clause)
+
+Documentation may be licensed under:
+  * [CC BY 4.0](http://creativecommons.org/licenses/by/4.0/)
+  * [MIT](https://opensource.org/licenses/MIT)
+
+If you need to use a license which isn't in the [IP Policy],
+you will need to obtain permission from the Board to do so.
+Please [open an issue in the CPC repository][new cpc issue],
```

Comment:
Where is `[new cpc issue]:` defined?

Do we need

    [new cpc issue]: https://github.com/openjs-foundation/cross-project-council/issues/new

at the bottom?

Reply:
```suggestion
Please [open an issue in the CPC repository](https://github.com/openjs-foundation/cross-project-council/issues/new),
```